api/tests/Feature/ApplicantFilterTest.php

```diff
@@ -35,7 +36,7 @@ class ApplicantFilterTest extends TestCase
     protected function setUp(): void
     {
         parent::setUp();
-
+        Notify::spy(); // don't send any notifications
```

Comment:
So is this something we should add to all Test classes now, to ensure we don't accidentally send notifications? Maybe we should add this to a base class.